src/graph.ts

```diff
@@ -24,6 +24,11 @@ function pathToName(path: string) {
     return name;
 }
 
+function escapeMermaid(str: string) {
+    const escapeChars = '___'
```

Comment:
why is this 3 times underscore?

Reply:
I am not sure what escape chars to use. So I thought 3 is rare enough.
Any suggestion?

Reply:
I doubt there will be conflicts, so readability is more important. So maybe one underscore should be fine

Reply:
Using one underscore.